### PR TITLE
Add lastUpdatedAt

### PR DESCRIPTION
**Pull request by ben-kaufman**

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -124,3 +125,11 @@ export function fixJsonQuotes(target: string): string {
      }
      return result;
 }
+
+export function save(ent: Entity | null, entName: string, timestamp: BigInt): void {
+  if (ent == null) {
+    return;
+  }
+  (ent as Entity).setBigInt('lastUpdatedAt', timestamp);
+  store.set(entName, ent.getString('id'), ent as Entity);
```
Comment (orenyodfat) [inline src/utils.ts:134]: does ent.save () will not do the work here ?

File: src/utils.ts [hunk 2]
```diff
@@ -124,3 +125,11 @@ export function fixJsonQuotes(target: string): string {
      }
      return result;
 }
+
+export function save(ent: Entity | null, entName: string, timestamp: BigInt): void {
```
Comment (orenyodfat) [inline src/utils.ts:129]: why it need to get entName ?

File: src/utils.ts (hunk 1 above)
Comment (ben-kaufman) [reply src/utils.ts:134]: No, it doesn't exist here

File: src/utils.ts (hunk 2 above)
Comment (ben-kaufman) [reply src/utils.ts:129]: Becuase  ent.save() doesn't exist

Comment (ben-kaufman) [reply src/utils.ts:129]: I know this need for the `entName ` is bad and very very error-prone, but I don't think there is a better way around it.

Comment (orenyodfat) [reply src/utils.ts:129]: we might missing the asserts that .save do ..
e.g 
`assert(id !== null, "Cannot save AvatarContract entity without an ID");
    assert(
      id.kind == ValueKind.STRING,
      "Cannot save AvatarContract entity with non-string ID. " +
        'Considering using .toHex() to convert the "id" to a string.'
    );`

Comment (ben-kaufman) [reply src/utils.ts:129]: I guess we could manually add them, but this is obviously not a very optimal solution...

**Pull request closed without merging**